radio/src/targets/common/arm/stm32/timers_driver.cpp

```diff
@@ -66,6 +66,11 @@ static void interrupt1ms()
   }
 #endif
 
+#if defined(SBUS_TRAINER)
+  sbusTrainerPauseCheck();
+  crsfTrainerPauseCheck();
+#endif
```

Comment:
As discussed, this needs eventually to be moved to a software timer.

Reply:
Yes, the next improvement of this PR will use soft-timers.